_includes/configuration/taborder.md

```diff
@@ -0,0 +1,6 @@
+## Tab order
+
+The tab order of the elements in a page, including {{site.productname}}, should be configured by setting the [`tabindex` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) on the relevant elements. The browser will then natively handle the tab order.
+
+To configure `tabindex` for the {{site.productname}} editor, set the attribute on the target element for the editor, or the equivalent if using a framework. 
```

Comment:
I'm not a big fan of "equivalent if using a framework" — target elements would still exist if you were using a framework, generally. The only time when people wouldn't be able to stick an attribute on a target element is if they're using one of our integrations that abstracts that away from them — is there a chance we could replace this with an explanation on how to change the `tabindex` using our integrations (hopefully by linking it to docs that have already been written for this)?

Reply:
Hm. I doubt we have docs about that, and I guess I added this because I wasn't sure what integrations would do. But I'd be fine with removing any mention of integrations.

Reply:
Removed the bit about integrations